File: app/src/main/java/com/worldtech/camera2video/utils/CommonUtil.java
package com.worldtech.camera2video.utils;

public class CommonUtil {

    private static long lastClickTime;

    /**
     * 是否过快点击
     */
    public static boolean fastClick() {
        long time = System.currentTimeMillis();
        long timeD = time - lastClickTime;
        if (0 < timeD && timeD < 800) {
            return true;
        }
        lastClickTime = time;
        return false;
    }

}

File: app/src/main/java/com/worldtech/camera2video/utils/CommonUtil.kt
package com.worldtech.camera2video.utils

object CommonUtil {
    private var lastClickTime: Long = 0

    /**
     * 是否过快点击
     */
    fun fastClick(): Boolean {
        val time = System.currentTimeMillis()
        val timeD = time - lastClickTime
        if (timeD in 1..799) {
            return true
        }
        lastClickTime = time
        return false
    }
}